### PR TITLE
Add a comment (Steal the code)

### PR DESCRIPTION
**Pull request by DogLoverPink**

### CODE REVIEW

File: src/main/java/frc/robot/QuickActions.java [hunk 1]
```diff
@@ -1,3 +1,5 @@
+// yoinky yoinky ahahahaha this code is mine now yippee!!1!
```
Comment (DogLoverPink) [inline src/main/java/frc/robot/QuickActions.java:1]: im gonna steel the code bc im cooler 😱

Comment (DogLoverPink) [reply src/main/java/frc/robot/QuickActions.java:1]: (this is a test PR guys pls dont accept)

Comment (julspr) [reply src/main/java/frc/robot/QuickActions.java:1]: ok i will accept it :)

**Review by julspr — COMMENTED** [review]
good

**Review by julspr — APPROVED** [review]

**Pull request merged**